vast/CMakeLists.txt

```diff
@@ -65,6 +65,7 @@ file(
       --set \"$base_dir\"/vast_integration_suite.yaml \
       \"$@\"
      $<$<BOOL:${VAST_ENABLE_LSVAST}>:
+     mkdir -p lsvast-integration-test/vast.db
```

Comment:
Why would this be necessary?

Reply:
VAST doesn't start if the directory doesn't exist:
```
[benno@L build]$ ./bin/vast -d foo/bar start
unable to create directory "foo/bar": No such file or directory
```

Reply:
So then it should probably just be `mkdir lsvast-integration-test`, because VAST will and should create the DB directory.